tests/test_requirements.txt

```diff
@@ -12,9 +12,9 @@ nose~=1.3.7
 pandas~=1.1.2
 pytest~=5.2
 requests~=2.20.0
-scipy~=1.3.2
+scipy~=1.5.4
```

Comment:
wouldn't `scipy~=1.5.3` solve our problem? It would pick up 1.5.3 on windows (which is avaialable) and 1.5.4 elsewhere?

Reply:
Good idea, let's try !

Reply:
tried both `scipy~=1.5.3` and `scipy~=1.5.0` and the _new_ problem is :
```
error: Cannot compile MPI programs. Check your configuration!!!
```